capstone/src/main/java/com/google/step/servlets/SeedDataServlet.java

```diff
@@ -0,0 +1,156 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.GeoPt;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that seeds some example content to datastore. */
+@WebServlet("/seedData")
+public class SeedDataServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    // Create entity and load with data
+    Entity restaurantInfo1 = new Entity("RestaurantInfo");
+    restaurantInfo1.setProperty("restaurantKey", 0);
+    restaurantInfo1.setProperty("name", "McDonald's");
+    restaurantInfo1.setProperty("location", new GeoPt((float) 42.297522, (float) -87.956039));
+    restaurantInfo1.setProperty(
+        "story", "We're a global business with the best french fries around.");
+    List<String> cuisineList1 = new ArrayList<>();
+    cuisineList1.add("American");
+    cuisineList1.add("Fast food");
+    restaurantInfo1.setProperty("cuisine", cuisineList1);
+    restaurantInfo1.setProperty("phone", "(847)362-3040");
```

Comment:
Do we have a plan for regulating the format of phone numbers?

Reply:
Looks like we can use the formatted_phone_number field returned by the Places API, which "contains the place's phone number in its local format" (https://developers.google.com/places/web-service/details). Then we can just put that string in the phone DOM element, if that sounds good.

Reply:
In the restaurantSignUpServlet I just made sure that the phone number was only numbers but didn't check beyond that.